src/core/components.py

```diff
@@ -342,6 +342,55 @@ def get_base_component_dict(self):
         }
 
 
+class Selection(Component):
+    __slots__ = [
+        "style",
+        "preconditions",
+        "destination_path",
+        "options_key",
+        "options_values",
+    ]
+
+    label = Translatable()
+
+    def get_options(self, options_key, options_values):
```

Comment:
we don't have a way to get_options without validation, if we don't think it'll ever be needed it's fine

Reply:
at the moment this would break the frontend implementation, if we ever want to do both options at the same time, we would need to re-engineer the solution